build.gradle

```diff
@@ -203,6 +203,11 @@ allprojects {
     options.links "https://docs.oracle.com/en/java/javase/${JavaVersion.current().majorVersion}/docs/api/"
   }
 
+  tasks.withType(Checkstyle) {
```

Comment:
I expect this is no longer required since the memory shortage was diagnosed and fixed separately.

Reply:
@AndrewJSchofield synced with trunk - the checkstyle change is no longer part of this PR